x-pack/plugins/apm/server/lib/helpers/get_aggregated_metrics.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { APMEventClient } from './create_es_client/create_apm_event_client';
+import { getSearchAggregatedTransactions } from './transactions';
+import { getSearchAggregatedServiceMetrics } from './service_metrics';
+import { APMConfig } from '../..';
+
+export async function getAggregatedMetrics({
+  config,
+  apmEventClient,
+  start,
+  end,
+  kuery,
+}: {
+  config: APMConfig;
+  apmEventClient: APMEventClient;
+  start?: number;
```

Comment:
Didn't change it. `start` and `end` were optional before

Reply:
This is a new function no? I don't quickly see us having to use it without the guarantee of start/end being set. We seemingly only use it for the service inventory where these are known to be set. If we can make things required, we should, it's easy to forget to pass certain props down and in this case the cost of not adding a range query can be quite high because ES has to go to all the shards to get an answer rather than being able to skip them based on their metadata.

Reply:
Now got what you mean. Will update them